### ISSUE TITLE
Sync Recent Emojis

### ISSUE THREAD

**Issue by TomasKostadinov** [threema-ch/threema-web#166]
## Expected Behavior
When opening the emoji panel the icons should display a "recent used" tab like in the app itself. 

## Current Behavior
The recent emoji tab is missing the webclient.

## Possible Solution
Maybe sync emojis on startup or whenever a new emoji gets used in-app.

## Your Environment
* Threema Web version: 1.0.5
* Threema for Android version: 3.1
* Threema for Android source: Play Store
* Operating system and version: Android 7.1.2 Developer Preview

**Comment by sillych** [comment]
What if a emoji used in the web client? should they also be synchronized to the app?

**Comment by lgrahl** [comment]
@sillych Yeah, my opinion is they should be synced bidirectional.

Another thing: Syncing Emojis may be slightly problematic due to different supported Unicode versions, see #317.

**Comment by rugk** [comment]
> Syncing Emojis may be slightly problematic due to different supported Unicode versions, 

Well… the web client and app could just skip emojis they don't know, when creating the "recently used" list.

**Comment by lgrahl** [comment]
Nah, it's not that easy if you sync **bidirectionally**. Without taking precautions you'd erase unknown emojis from each other's lists.
